Clarifying anycast vs local prefixes (#19221)
cloudflare/cloudflare-docs: src/content/docs/network-interconnect/classic-cni/set-up/configure-bgp-bfd.mdx

After you provision the IPs and the ping tests confirm the connection, accept the routes from the BGP session Cloudflare configured. Configuring the BGP session on both the Cloudflare and user sides requires a BGP call and an approximately two hour maintenance window that you provide to Cloudflare.

> Cloudflare advertises all of its available anycast prefixes including [BYOIP](/byoip/) prefixes over CNI, but this can exclude other location-specific prefixes that may not be available at that datacenter, and the BGP session occurs over a private link from Cloudflare to the customer data center.

Reviewer: ```suggestion
Cloudflare advertises all of its available anycast prefixes, including [BYOIP](/byoip/) prefixes, over a private CNI link between Cloudflare and the customer data center. However, this can exclude other location-specific prefixes that may not be available at that datacenter.
```
